jsr352/core/src/test/java/org/hibernate/search/jsr352/massindexing/MassIndexingJobParametersBuilderTest.java

```diff
@@ -115,4 +120,25 @@ public void testRestrictedBy_twoRestrictionTypes() {
 				.restrictedBy( "from string" )
 				.restrictedBy( Restrictions.isEmpty( "dummy" ) );
 	}
+
+	@Test
+	public void testTenantId_null() {
+		try {
+			MassIndexingJob.parameters().forEntity( String.class ).tenantId( null );
+		}
+		catch (NullPointerException e) {
+			assertEquals( e.getMessage(), "Your tenantId is null, please provide a valid tenant ID.", e.getMessage() );
```

Comment:
This test will pass if the method doesn't throw an exception, which is not what you want. When checking whether code throws an exception, you'll probably find it easier to use an `ExpectedException` rule. You can have a look at `org.hibernate.search.test.util.impl.LoggingCreationTest` to see how to use it.